python/pyarrow/_dataset.pyx

```diff
@@ -2435,7 +2462,9 @@ cdef class Scanner(_Weakrefable):
         builder = make_shared[CScannerBuilder](pyarrow_unwrap_schema(schema),
                                                fragment.unwrap(), options)
         _populate_builder(builder, columns=columns, filter=filter,
-                          batch_size=batch_size, use_threads=use_threads,
+                          batch_size=batch_size, batch_readahead=batch_readahead,
+                          fragment_readahead=_DEFAULT_FRAGMENT_READAHEAD,
```

Comment:
```suggestion
```
I don't think we need to specify this kwarg if we're just going to specify the default.

Reply:
This is a Cython quirk. You have to specify all the arguments.